src/coreclr/src/vm/clrtracelogging.cpp

```diff
@@ -1,39 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-//*****************************************************************************
-// clrttracelogging.cpp
```

Comment:
This tracelogging is also used by src\coreclr\src\jit\jittelemetry.cpp. I think it can be deleted as well.

Reply:
Looks like telemetry is used by the JIT (in `ee_il_dll.cpp`, `NotifyDllProcessAttach` is used).

Reply:
`FEATURE_TRACELOGGING` is never defined and everything under it is unused code. It is spread over multiple files - may be best to submit separate PR to clean it up.

Reply:
Makes sense. I will followup on `FEATURE_TRACELOGGING`, `FEATURE_METADATA_PERF_STATS` and `PAL_PERF` separately.